gamemodes/terrortown/gamemode/shared/lang/italian.lua

```diff
@@ -639,7 +639,7 @@ L.karma_min = "Irresponsabile"
 L.corpse = "Cadavere"
 L.corpse_hint = "Premi {usekey} per identificare. {walkkey} + {usekey} per identificare segretamente."
 
-L.target_disg = "(TRAVESTITO)"
+L.target_disg = "(travestito)"
```

Comment:
Those brackets shouldn't be a part of the translation

Reply:
They were before and I though it was cheaper that adding two string concats each frame

Reply:
Hmm for sure, that's true, but anyways that's not a part of a translation by default...

Reply:
I can add the brackets in the rendering hook if you like that more